Currently rdp_event.c.o can not be built

Do I miss a library here? Any hint?

```
[   70s] [ 95%] Linking C shared module remmina-plugin-rdp.so

[   70s] cd /home/abuild/rpmbuild/BUILD/Remmina-1.1.99.git.20160117/build/remmina-plugins/rdp && /usr/bin/cmake -E cmake_link_script CMakeFiles/remmina-plugin-rdp.dir/link.txt --verbose=1

[   70s] /usr/bin/cc  -fPIC -fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector -funwind-tables -fasynchronous-unwind-tables -DNDEBUG -Wall -fPIC -Wno-unused-result -Wno-unused-but-set-variable -Wno-unused-function -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now -shared  -o remmina-plugin-rdp.so CMakeFiles/remmina-plugin-rdp.dir/rdp_plugin.c.o CMakeFiles/remmina-plugin-rdp.dir/rdp_event.c.o CMakeFiles/remmina-plugin-rdp.dir/rdp_file.c.o CMakeFiles/remmina-plugin-rdp.dir/rdp_settings.c.o CMakeFiles/remmina-plugin-rdp.dir/rdp_gdi.c.o CMakeFiles/remmina-plugin-rdp.dir/rdp_graphics.c.o CMakeFiles/remmina-plugin-rdp.dir/rdp_cliprdr.c.o CMakeFiles/remmina-plugin-rdp.dir/rdp_channels.c.o -lgtk-3 -lgdk-3 -lglib-2.0 -lgio-2.0 -lgobject-2.0 -lgmodule-2.0 -lgthread-2.0 -lpango-1.0 -lcairo -lgdk_pixbuf-2.0 -latk-1.0 -lpthread -lfreerdp -lfreerdp-client -lwinpr 
[   70s] CMakeFiles/remmina-plugin-rdp.dir/rdp_event.c.o: In function `remmina_rdp_event_on_key':
[   70s] rdp_event.c:(.text+0x796): undefined reference to `XkbKeycodeToKeysym'
[   70s] rdp_event.c:(.text+0x7ac): undefined reference to `XKeysymToKeycode'
[   70s] collect2: error: ld returned 1 exit status
[   70s] remmina-plugins/rdp/CMakeFiles/remmina-plugin-rdp.dir/build.make:294: recipe for target 'remmina-plugins/rdp/remmina-plugin-rdp.so' failed

--- a/remmina-plugins/rdp/CMakeLists.txt
+++ b/remmina-plugins/rdp/CMakeLists.txt
@@ -34,6 +34,7 @@
 
 set(CMAKE_THREAD_PREFER_PTHREAD)
 find_package(Threads)
+find_package(X11)
 
 set(REMMINA_PLUGIN_RDP_SRCS
 	rdp_plugin.c
@@ -62,8 +63,8 @@ add_library(remmina-plugin-rdp MODULE ${REMMINA_PLUGIN_RDP_SRCS})
 set_target_properties(remmina-plugin-rdp PROPERTIES PREFIX "")
 set_target_properties(remmina-plugin-rdp PROPERTIES NO_SONAME 1)
 
-include_directories(${REMMINA_COMMON_INCLUDE_DIRS} ${FREERDP_INCLUDE_DIRS})
-target_link_libraries(remmina-plugin-rdp ${REMMINA_COMMON_LIBRARIES} ${FREERDP_LIBRARIES})
+include_directories(${REMMINA_COMMON_INCLUDE_DIRS} ${FREERDP_INCLUDE_DIRS} ${X11_INCLUDE_DIR})
+target_link_libraries(remmina-plugin-rdp ${REMMINA_COMMON_LIBRARIES} ${FREERDP_LIBRARIES} ${X11_LIBRARIES})
 
 install(TARGETS remmina-plugin-rdp DESTINATION ${REMMINA_PLUGINDIR})
 

